hub/core/tests/common.py

```diff
@@ -27,3 +28,13 @@
     ALL_PROVIDERS + ALL_CACHES,  # type: ignore
     indirect=True,
 )
+
```

Comment:
missing `parametrize_all_dataset_caches`

Reply:
Not gonna add it till I need it. ;)